src/forms/FormBuilder.js

```diff
@@ -127,10 +126,11 @@ export default class FormBuilder extends Component {
   renderPreview() {
     if(!this.props.preview) return null;
 
-    const form = Object.assign({}, this.props.forms.form);
```

Comment:
Not familiar with this part of the code, just note that this was copying the object before. Maybe not needed.